sources/api/netdog/src/cli/install.rs

```diff
@@ -52,37 +50,23 @@ pub(crate) fn run(args: InstallArgs) -> Result<()> {
     }
 
     match (&args.interface_type, &args.interface_family) {
-        (InterfaceType::Dhcp, InterfaceFamily::Ipv4) => {
-            let info =
-                LeaseInfo::from_lease(&args.data_file).context(error::LeaseParseFailedSnafu {
-                    path: &args.data_file,
-                })?;
-            // Randomize name server order, for libc implementations like musl that send
-            // queries to the first N servers.
-            let mut dns_servers: Vec<_> = info.dns_servers.iter().collect();
-            dns_servers.shuffle(&mut thread_rng());
-            write_resolv_conf(&dns_servers, &info.dns_search)?;
-            write_current_ip(&info.ip_address.addr())?;
-        }
-        _ => eprintln!("Unhandled 'install' command: {:?}", &args),
```

Comment:
The compiler was yelling at me to remove this line... I kept the `match` syntax here as it felt the most clean.  I also didn't want to throw a grenade into the current arg parsing since failing to parse these args means the helpers fail.